pkg/operator/operator.go

```diff
@@ -180,6 +191,20 @@ func (o *Operator) Start(stop <-chan struct{}) error {
 		log.Info("cache synced")
 	}
 
+	// Sync the CA cert every once in a while.
+	ticker := time.NewTicker(24 * time.Hour)
+	go func() {
+		for {
+			select {
+			case <-stop:
+				return
+			case <-ticker.C:
+				ci := ingressv1alpha1.ClusterIngress{}
```

Comment:
Not sure if this particular type makes sense for the event, but the controller doesn't care, and this is convenient